resources/styles/components/task-step/ends.less

```diff
@@ -0,0 +1,5 @@
+.task-review-completed{
+  .panel-footer {
+    display: none;
```

Comment:
When you include a `display: none;` can you include a comment why? When revisiting the code it seems odd to have written code to generate the DOM and then hide it.